<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tworzenie_konta</class>
 <widget class="QDialog" name="Tworzenie_konta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zakładanie konta</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Podaj Dane </string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="imie_Edit"/>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="potwierdz_Button">
        <property name="text">
         <string>Potwierdź</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="email_Edit"/>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="haslo_Edit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="nazwisko_Edit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="imie_label">
        <property name="text">
         <string>Imie:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Hasło:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>192</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nazwisko:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Powtórz hasło:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="powtorz_haslo_Edit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
